klab.hub/src/main/java/org/integratedmodelling/klab/hub/users/services/UserGroupEntryServiceImpl.java

```diff
@@ -17,6 +17,7 @@
 import org.integratedmodelling.klab.hub.payload.UpdateUsersGroups;
 import org.integratedmodelling.klab.hub.repository.MongoGroupRepository;
 import org.integratedmodelling.klab.hub.repository.UserRepository;
+import org.joda.time.DateTime;
```

Comment:
Shouldn't we use `java.time.*` instead of `joda`?

Reply:
It seems an unused import, safe to delete
We have removed the `joda` dependency from the `pom.xml`